integrations/java/openhouse-java-runtime/src/main/java/com/linkedin/openhouse/javaclient/OpenHouseCatalog.java

```diff
@@ -93,32 +95,15 @@ public class OpenHouseCatalog extends BaseMetastoreCatalog
 
   private static final String HTTP_CONNECTION_STRATEGY = "http-connection-strategy";
 
+
   @Override
   public void initialize(String name, Map<String, String> properties) {
     this.name = name;
     this.properties = properties;
-    String uri = properties.get(CatalogProperties.URI);
```

Comment:
Can we revert this refactor and do it in a future PR?

Reply:
Yes, please revert. As you point out, the ask is directed towards keeping changes limited to requirements of tokenRefetch. Not sure if this refactor is core to it.

Reply:
Done, reverted to refactoring. Will being them in after the token refetch is merged.